public/src/components/trading_terminal/OpenOrders.tsx

```diff
@@ -279,7 +280,7 @@ export class OpenOrders extends React.Component<OpenOrdersProps, OpenOrdersState
 					<td className="red volume"><strong>{openAsks[askId].getStockQuantity()}</strong></td>
 					<td className="red volume"><strong>{openAsks[askId].getStockQuantityFulfilled()}</strong></td>
 					<td className="red volume"><strong>{price}</strong></td>
-					<td onClick={e => this.confirmCancelModal(this,askId,true)} className="red cancel-order-button">❌</td>
+					 <td><button onClick={e => this.confirmCancelModal(this,askId,true)} disabled={this.props.isMarketOpen ? false : true} className="red cancel-order-button">❌</button></td>
```

Comment:
Instead of writing `this.props.isMarketOpen ? false : true` thrice, define a variable that stores it.